assets/stylesheets/screens/_session.scss

```
@@ -0,0 +1,27 @@
+.session.get,
+.session.post {
+  .session-wrapper {
+    @apply flex flex-col;
+  }
+  .session-card {
```

Prefer using the component as prefix

```suggestion
  .card-session {
```

The same applied below for `form-session

Please refer to Compass:
> Use the UI element type as a prefix following by one to two words. While it might seem counter-intuitive at first, this technique makes it easier to differentiate component purpose and search through components. Most framework use this technique e.g. btn-primary is not called primary-btn as btn is the namespace grouping all buttons.
> https://nimblehq.co/compass/development/code-conventions/css/#class-names